packages/ui/src/app/pages/Forum/ForumCategory.tsx

```diff
@@ -73,6 +73,11 @@ export const ForumCategory = () => {
               <PageTitle>{category.title}</PageTitle>
             </PreviousPage>
           }
+          description={
+            <TextMedium className="category-description" normalWeight inter lighter truncateLines={2}>
```

Comment:
@traumschule perhaps "truncateLines" should be  removed to show description text

Reply:
=> #4070